### ISSUE TITLE
Why can the model correctly predict empty occupancies?

### ISSUE THREAD

**Issue by lnSong** [weiyithu/SurroundOcc#122]
Thank you for your work! 
The code occ[..., 3][occ[..., 3] == 0] = 255 sets empty occupancies to 255, 
and the loss criterion is defined as:
criterion = nn.CrossEntropyLoss(  
    ignore_index=255, reduction="mean"  
)  
Since the loss ignores the index 255, why is the model still able to correctly infer empty occupancies?

**Comment by lnSong** [comment]
non-occupied is trained by geo_scal_loss
